Resources/Locale/en-US/chemistry/components/injector-component.ftl

```diff
@@ -27,3 +27,4 @@ injector-component-drawing-user = You start drawing the needle.
 injector-component-injecting-user = You start injecting the needle.
 injector-component-drawing-target = {CAPITALIZE(THE($user))} is trying to use a needle to draw from you!
 injector-component-injecting-target = {CAPITALIZE(THE($user))} is trying to inject a needle into you!
+injector-component-deny-user = Exoskeleton too thick!
```

Comment:
It's hardcoded here for Chitinids, but should instead be a generic system where the component defines which localization it uses. It could also be extended to use the target's name.

```suggestion
injector-component-deny-chitinid = {CAPITALIZE(($target))}'s exoskeleton is too thick for the needle to pierce.
```